sdk/src/main/java/com/meniga/sdk/models/budget/MenigaBudget.java

```diff
@@ -166,16 +168,22 @@ public MenigaBudget createFromParcel(Parcel source) {
 		}
 	};
 
-	/*
-	* API Calls
```

Comment:
I left this comment block as a visual guide for me when quickly browsing the classes to note where the api calls start (always towards the bottom of the file) vs the normal class stuff like getters, equals, parcelable etc.